test/inductor/test_cpu_select_algorithm.py

```diff
@@ -1569,6 +1569,105 @@ def forward(self, x):
             self.common(mod, (v,), atol=atol, rtol=rtol)
         self.assertEqual(counters["inductor"]["select_algorithm_autotune"], 1)
 
+    @inductor_config.patch({"freezing": False})
+    @patches
+    @torch.no_grad
+    @unittest.skipIf(not TEST_MKL, "Test requires MKL")
+    @parametrize("batch_size", (16,))
+    @parametrize("in_features", (128,))
+    @parametrize("out_features", (64,))
+    @parametrize("bias", (True,))
+    @dtypes(
+        torch.float,
+    )
+    def test_aoti_linear(self, batch_size, in_features, out_features, bias, dtype):
+        try:
+            from .test_aot_inductor_utils import AOTIRunnerUtil
+        except ImportError:
+            # skip this UT if relative import failed
```

Comment:
Why would this fail?

Reply:
Yes, it fails at some CI jobs. I found the other UT code about `relative import` has same workaround as in https://github.com/pytorch/pytorch/blob/66340e67515cd3592bda6bdd9bfe2ffa22fe7413/test/inductor/test_cpu_repro.py#L41-L49

Reply:
But can we do `import test_aot_inductor_utils` on import failures?

Reply:
sure, I will add it as the other code did